hasher-matcher-actioner/terraform/counters/variables.tf

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+variable "region" {
+  description = "AWS region to deploy to"
+  type        = string
+  default     = "us-east-1"
+}
+
+variable "profile" {
+  description = "AWS profile to use for authentication"
+  type        = string
+  default     = null
+}
+
+variable "prefix" {
+  description = "Prefix to use for resource names"
+  type        = string
+}
+
+variable "lambda_docker_info" {
+  description = "Docker container information for lambda functions"
+  type = object({
+    uri = string
+    commands = object({
+      match_counter = string
+    })
+  })
+}
+
+variable "matches_sns_topic_arn" {
+  description = "ARN for the topic that collects matches from matchers."
+  type        = string
+}
+
+variable "log_retention_in_days" {
+  description = "How long to retain cloudwatch logs for lambda functions in days"
+  type        = number
+}
+
+variable "datastore" {
+  description = "DynamoDB Table to store hash and match information into"
+  type = object({
+    name = string
+    arn  = string
+  })
+}
+
+variable "config_table" {
```

Comment:
Could be missing something but it doesn't look like this or anything to do with the "config_table" is used but all are passed in. Could be from a copy paste and I think it is likely we will want to add it eventually to enable configurations after tf apply but could you note it as currently unused  in a comment if you want to keep it in the meantime? 

(my [not blocking] preference is to reduce the amount of things we give to what is being used for now)